general_tools/file_utils.py

```diff
@@ -99,8 +99,8 @@ def load_json_object(file_name, default=None):
 
 
 def read_file(file_name):
-    contents = codecs.open(file_name, 'r', encoding='utf-8').read()
-    return contents
+    with codecs.open(file_name, 'r', encoding='utf-8') as f:
```

Comment:
Updated. I've added an optional `encoding` parameter which defaults to `'utf-8-sig'`.